onchain/rollups/hardhat.config.ts

```diff
@@ -117,11 +121,17 @@ const config: HardhatUserConfig = {
             arbitrum_goerli: [
                 ppath("@cartesi/util", "/deployments/arbitrum_goerli"),
             ],
+            arbitrum_sepolia: [
+                ppath("@cartesi/util", "/deployments/arbitrum_sepolia"),
+            ],
             mainnet: [ppath("@cartesi/util", "/deployments/mainnet")],
             optimism: [ppath("@cartesi/util", "/deployments/optimism")],
             optimism_goerli: [
```

Comment:
I would remove Goerli from the config since its deprecated.

Reply:
We agreed to remove only on the next major, to not be a breaking change